sql/moz-fx-data-shared-prod/pocket/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Pocket Views
+description: |-
+  Views related to Pocket data
```

Comment:
Currently the autogenerated metadata in https://github.com/mozilla/private-bigquery-etl/blob/private-generated-sql/sql/moz-fx-data-shared-prod/pocket/dataset_metadata.yaml differs from this in an inconsequential way. Note for situational awareness that we're going to be deleting the pocket namespace in https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/main/schemas/pocket/fire-tv-events via https://mozilla-hub.atlassian.net/browse/DSRE-1830.

Reply:
Thanks for the update!

In future, before we apply missing metadata settings for a dataset, do you recommend we verify if its setup in other places (example: [private-bigquery-etl](https://github.com/mozilla/private-bigquery-etl) repo) before making such updates?